mvds/sync-modes.md

```diff
@@ -0,0 +1,14 @@
+#  MVDS Sync Modes
+
+> Version: 0.0.1 (Draft)
+> 
+> Authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
+
+##  Table of Contents
+
+1. [Abstract](#abstract)
+
+## Abstract
+
+In this specification, we describe a method to provide consistency through various means as well as modifying synchronization of [MVDS](./README.md).
```

Comment:
Maybe explain options laid out here, not jsut consistency

Reply:
Does that really need to be detailed in the abstract @oskarth